internal/fsutil.go

```diff
@@ -72,6 +73,55 @@ func createTestExtImg(ctx context.Context, t *testing.T, extName string, testFil
 	return imgFile.Name()
 }
 
+// DeviceFile represents a block device
+type DeviceFile struct {
+	Subpath string
+	UID     int
+	GID     int
+	Dev     int // major number
+}
+
+// CreateBlockDevice creates a block device, or block special file for testing
+func CreateBlockDevice(ctx context.Context, t *testing.T, fsType string, testFile DeviceFile) string {
+	t.Helper()
+
+	switch fsType {
+	case "ext3", "ext4":
```

Comment:
do we still use/support ext3?

Reply:
@kzys What do you think? Should we change both **CreateFSImg** and **CreateBlockDevice** to only include **ext4**?
I keep `ext3` to be consistent with [CreateFSImg](https://github.com/firecracker-microvm/firecracker-containerd/blob/c2323bc71886b3abfefd9afa53244740f70db0b8/internal/fsutil.go#L32).

Reply:
I think getting rid of ext3 is fine.

Reply:
Fixed.